src/main/groovy/io/github/cosmicsilence/scalafix/ScalafixPlugin.groovy

```diff
@@ -33,12 +34,12 @@ class ScalafixPlugin implements Plugin<Project> {
         def customRulesConfiguration = project.configurations.create(CUSTOM_RULES_CONFIGURATION)
         customRulesConfiguration.description = "Dependencies containing custom Scalafix rules"
 
-        configureTasks(project, extension)
-
         project.afterEvaluate {
             if (extension.autoConfigureSemanticdb) {
                 configureSemanticdbCompilerPlugin(project)
             }
+
+            configureTasks(project, extension)
```

Comment:
now that compiler options are passed into the task class as input, we need to configure the semanticdb plugin before we run `configureTasks`